### PR TITLE
Caps stock price at 1000

### PR DESCRIPTION
**Pull request by Iamgoofball**
This wont actually fix the problem of the crew having free emags @KorPhaeron but whatever, after this is merged it'll be proven I'm right here

### CODE REVIEW

**Comment by Spaceinaba** [comment]
now all we have to do is wait for somebody to just replace it with something actually fun to do (HURRY UP DOOM YOU DORK)

**Comment by Bombpants** [comment]
How high was it going before this? 1000 seems a little low (cargo memes best memes)

**Comment by MrDoomBringer** [comment]
why dont you fix the shitty rounding error causing gay stocks instead of a hardcoded cap?

**Comment by Tlaltecuhtli** [comment]
@Bombpants http://take.ms/qbevF

**Comment by ExcessiveUseOfCobblestone** [comment]
@Bombpants if your stock is going 1000 per, something is off. With that said, I think the fix isn't an actual fix (although mentioned in OP).

Most of the time 3 companies fluctuate in between 100-200 while the others are lower end fluctuating from ~15-<100 (per). Your profit is basically buying as much as you can when they hit rock bottom and decide to miraculously rebound, which is pretty much guaranteed.

In hindsight the way stock is used to profit is exploiting the fact the system is godawful. I wouldn't be terribly upset with its removal until I get bootyblasted enough to force myself to either review or rewrite the thing.

**Comment by Iamgoofball** [comment]
@MrDoomBringer if you can figure out where the actual error is go for the fix please

**Comment by pigeonsk** [comment]
Your spike detection had a useless conditional what the fuck was that even for.

>low number
>high number
>if (x < high number OR x > low number)

???????????????????????????????????????????????????

**Comment by pigeonsk** [comment]
Also if there is a balance issue could someone please document it in a repo issue? I see bits of references to such cargo problems here and there but no aggregated documentation whatsoever means neither I nor anyone else can come up with a holistic and wholesome solution and we end up with garbage band-aid fixes like this PR instead.

File: code/modules/stock_market/stocks.dm [hunk 1]
```diff
@@ -146,7 +146,8 @@
 		current_value += i_hate_this_code
 	if (current_value < 5)
 		current_value = 5
-
+	else if (current_value > 1000)
```
Comment (Jalleo) [inline code/modules/stock_market/stocks.dm]: Is there also a cap to prevent it going beyond -1000 ?

Comment (ExcessiveUseOfCobblestone) [reply code/modules/stock_market/stocks.dm]: I've never seen it go negative although i've seen the company die.

**Pull request closed without merging**